src/anupSJun20/MissingNumbrinIntegerArray.java

```diff
@@ -0,0 +1,20 @@
+package anupSJun20;
+/*Program 1: Find the missing number in integer array of 1 to 10?
+Hint: You have given an integer array which contains numbers from 1 to 10 but one number is missing, 
+you need to write a Java program to find that missing number in an array.*/
+
+public class MissingNumbrinIntegerArray {
+	static int findTheMissingNumber(int[] num,int startNum,int endNum) {
```

Comment:
**// Suggestion -**
Here, in entire method you are not using startNum integer(neither in calculation or in any SOP statements). So there is no need to pass it in method argument and you can reduce the argument as **static int findTheMissingNumber(int[] num, int endNum)**

Another way is, just pass input array in method argument- **static int findTheMissingNumber(int[] num)** and you can use **int endNum = array.length+1**